lib/package-detail-view.js

```diff
@@ -372,7 +372,16 @@ export default class PackageDetailView {
       readme = fs.readFileSync(this.readmePath, {encoding: 'utf8'})
     }
 
-    const readmeView = new PackageReadmeView(readme)
+    let readmeSrcBase
+
+    if (this.pack.path) {
+      readmeSrcBase = this.pack.path
+    } else {
+      let repoUrl = this.packageManager.getRepositoryUrl(this.pack)
+      readmeSrcBase = repoUrl + '/blob/master/'
```

Comment:
This needs to be changed to point to the last published tag.

Reply:
Do you know if there's a simple way to do that?

Reply:
I looked around a bit but it doesn't look like atom.io's API has a way of getting the latest tag.  There's also the problem that `master` isn't guaranteed to exist (and be the default branch).  Let me see if anyone else has any ideas.